### PR TITLE
OCPQE-14713: HyperShift: add health-check

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-p1-f7

File: ci-operator/step-registry/cucushift/hypershift-extended/enable-qe/pull-secret/cucushift-hypershift-extended-enable-qe-pull-secret-ref.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ ref:
   from: upi-installer
   cli: latest
   grace_period: 10m
-  timeout: 55m0s
+  timeout: 60m0s
```
Comment (heliubj18) [inline ci-operator/step-registry/cucushift/hypershift-extended/enable-qe/pull-secret/cucushift-hypershift-extended-enable-qe-pull-secret-ref.yaml:6]: why from 55 to 60? 55 is timeout?

File: ci-operator/step-registry/cucushift/hypershift-extended/enable-qe/catalogsource/cucushift-hypershift-extended-enable-qe-catalogsource-commands.sh [hunk 2]
```diff
@@ -85,14 +85,6 @@ EOF
...
         run_command "oc -n openshift-marketplace get pods -l olm.catalogSource=qe-app-registry -o yaml"
-        node_name=$(oc -n openshift-marketplace get pods -l olm.catalogSource=qe-app-registry -o=jsonpath='{.items[0].spec.nodeName}')
-        run_command "oc create ns debug-qe -o yaml | oc label -f - security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/audit=privileged pod-security.kubernetes.io/warn=privileged --overwrite"
-        run_command "oc -n debug-qe debug node/${node_name} -- chroot /host podman pull --authfile /var/lib/kubelet/config.json ${index_image}"
-
-        run_command "oc get mcp,node"
-        run_command "oc get mcp worker -o yaml"
-        run_command "oc get mc $(oc get mcp/worker --no-headers | awk '{print $2}') -o=jsonpath={.spec.config.storage.files}|jq '.[] | select(.path==\"/var/lib/kubelet/config.json\")'"
-
         return 1
```
Comment (heliubj18) [inline ci-operator/step-registry/cucushift/hypershift-extended/enable-qe/catalogsource/cucushift-hypershift-extended-enable-qe-catalogsource-commands.sh:88]: what's the purpose of these change?

Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/hypershift-extended/enable-qe/catalogsource/cucushift-hypershift-extended-enable-qe-catalogsource-commands.sh:88]: The function of this code is: the installation fails to print the log, but for hypershift guest cluster, it don't have mcp, so we should remove them

File: ci-operator/step-registry/cucushift/hypershift-extended/enable-qe/pull-secret/cucushift-hypershift-extended-enable-qe-pull-secret-ref.yaml (hunk 1 above)
Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/hypershift-extended/enable-qe/pull-secret/cucushift-hypershift-extended-enable-qe-pull-secret-ref.yaml:6]: yes, The maximum execution time of this step is 60m
